module/core/clone_dyn_meta/Cargo.toml

```diff
@@ -17,6 +17,9 @@ Derive to clone dyn structures.
 categories = [ "algorithms", "development-tools" ]
 keywords = [ "fundamental", "general-purpose" ]
 
+[package.metadata]
+stable_status = "experimental"
```

Comment:
stability not stable_status

Reply:
also it's optional thing
algorithm should works fine without the field
default is experimental